pkgs/development/python-modules/colcon-cargo/default.nix

```diff
@@ -0,0 +1,55 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  colcon,
+  setuptools,
+  pythonOlder,
+  pytestCheckHook,
+  pytest-cov-stub,
+  pytest-repeat,
+  pytest-rerunfailures,
+  toml,
+}:
+buildPythonPackage rec {
+  pname = "colcon-cargo";
+  version = "0.1.3";
+
+  disabled = pythonOlder "3.10";
+
+  src = fetchFromGitHub {
+    owner = "colcon";
+    repo = "colcon-cargo";
+    rev = version;
+    hash = "sha256-Do8i/Z1nn8wsj0xzCQdSaaXoDf9N34SiMb/GIe4YOs4="; # Replace with actual hash
+  };
+  build-system = [ setuptools ];
+
+  dependencies = [
+    colcon
+    toml
+  ];
+  nativeCheckInputs = [
+    pytestCheckHook
+    pytest-cov-stub
+    pytest-repeat
+    pytest-rerunfailures
+  ];
+  pytestFlagsArray = [
+    "--ignore=test/test_flake8.py"
+    "--ignore=test/test_spell_check.py"
+  ];
+  # The package doesn't have tests
+  doCheck = true;
+  pythonImportsCheck = [
+    "colcon_cargo"
+  ];
+
+  meta = {
+    description = "An extension for colcon-core to support Rust packages built with Cargo";
+    homepage = "https://github.com/colcon/colcon-cargo";
+    license = lib.licenses.asl20;
+    maintainers = with lib.maintainers; [ guelakais ]; # Add yourself if you're maintaining this
+    platforms = lib.platforms.all;
```

Comment:
I suspect you can remove this line as well:
https://github.com/NixOS/nixpkgs/blob/5099db4debb6ea7d6f12fd3f6d9fe2284b1ede81/pkgs/development/interpreters/python/mk-python-derivation.nix#L431